### DISCUSSION TITLE
输出文件中奇怪的`:local`

### DISCUSSION THREAD

**Discussion by p-liar** [umijs/umi#10983] [General]
我使用getLocalIdent 自定义类名，然后让'haha.less'这个文件不使用css Modules，但是这个文件里面 animation 使用缩写会被加一个奇怪的:local。
配置文件：
<img width="858" alt="image" src="https://user-images.githubusercontent.com/128342039/232479338-60668bde-fb0f-4fc4-8edb-ebaaa98b8605.png">
业务代码
<img width="962" alt="image" src="https://user-images.githubusercontent.com/128342039/232479558-0bf3de69-bd6f-4c60-a634-8a7de99dade4.png">
<img width="916" alt="image" src="https://user-images.githubusercontent.com/128342039/232479712-f7c11a1d-7b00-4ad1-abcb-246700fc1fda.png">
编译后的代码
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/128342039/232479985-4d03f75d-a72e-4b64-ba15-91c79a9a4274.png">

**Comment by fz6m** [comment]
umi 3 不维护了建议升级到 umi 4
